public/grpc-client.js

```diff
@@ -115,6 +146,14 @@ module.exports.init = async function({
     lnd[method](body, { deadline }, handleResponse);
   });
 
+  ipcMain.on('autopilotRequest', (event, { method, body = {} }) => {
+    const deadline = new Date(new Date().getTime() + GRPC_TIMEOUT);
+    const handleResponse = (err, response) => {
+      event.sender.send(`autopilotResponse_${method}`, { err, response });
+    };
+    autopilot[method](body, { deadline }, handleResponse);
+  });
+
```

Comment:
Seems like a pretty diff for just toggling autopilot. What's the advantage over restarting lnd with autopilot args disabled?

Reply:
If we restart `lnd`, the user will have to input their password again.

It's a pretty big diff, but we'll need most of the diff (the new protos, new autopilot methods, etc)  to implement the bos score feature anyway.